opm/common/data/SimulationDataContainer.hpp

```diff
@@ -63,6 +63,15 @@ namespace Opm {
         std::vector<double>& getFaceData( const std::string& name );
         const std::vector<double>& getFaceData( const std::string& name ) const;
 
+        /// Will return the number of components of the celldata with
+        /// name @name:
+        ///
+        ///    cellDataComponents( "PRESSURE" )   -> 1
+        ///    cellDataComponents( "SATURATION" ) -> 3 
+        ///
+        /// for a three phase model.
+        size_t cellDataComponents( const std::string& name ) const;
```

Comment:
I think this name is a little confusing, and would prefer numCellDataComponents(). Without the `num` I'd think it somehow should return the components themselves.